src/Examples/Flowsheets/HDA_flowsheet_with_distillation_solution_testing.ipynb

```diff
@@ -0,0 +1,1773 @@
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "exercise"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Todo: Add constraint: Outlet pressure = Inlet pressure\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "solution"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Todo: Add constraint: Outlet pressure = Inlet pressure\n",
+    "m.fs.translator.eq_pressure = Constraint(\n",
+    "    expr=m.fs.translator.outlet.pressure[0] ==\n",
+    "    m.fs.translator.inlet.pressure[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Remaining constraints on the translator block\n",
+    "\n",
+    "# Add constraint: Benzene mole fraction definition\n",
+    "m.fs.translator.eq_mole_frac_benzene = Constraint(\n",
+    "    expr=m.fs.translator.outlet.mole_frac_comp[0, \"benzene\"] ==\n",
+    "    m.fs.translator.inlet.flow_mol_phase_comp[0, \"Liq\", \"benzene\"] /\n",
+    "    (m.fs.translator.inlet.flow_mol_phase_comp[0, \"Liq\", \"benzene\"] +\n",
+    "    m.fs.translator.inlet.flow_mol_phase_comp[0, \"Liq\", \"toluene\"]))\n",
+    "\n",
+    "# Add constraint: Toluene mole fraction definition\n",
+    "m.fs.translator.eq_mole_frac_toluene = Constraint(\n",
+    "    expr=m.fs.translator.outlet.mole_frac_comp[0, \"toluene\"] ==\n",
+    "    m.fs.translator.inlet.flow_mol_phase_comp[0, \"Liq\", \"toluene\"] /\n",
+    "    (m.fs.translator.inlet.flow_mol_phase_comp[0, \"Liq\", \"benzene\"] +\n",
+    "    m.fs.translator.inlet.flow_mol_phase_comp[0, \"Liq\", \"toluene\"]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Inline Exercise:</b>\n",
+    "Finally, let's add the Heater H102 in the same way as H101 but pass the m.fs.bt_properties thermodynamic package. We will add the distillation column after converging the flowsheet.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "exercise"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Todo: Add the Heater H102 to the flowsheet\n",
+    " "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "solution"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Todo: Add the Heater H102 to the flowsheet\n",
+    "m.fs.H102 = Heater(default={\"property_package\": m.fs.bt_properties,\n",
+    "                            \"has_pressure_change\": True,\n",
+    "                            \"has_phase_equilibrium\": True})   "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connecting Unit Models using Arcs\n",
```

Comment:
It is probably worth reminding the user of the issue with the SD Tool here and hence that we will add the distillation column later.

Reply:
I have mentioned this just a few lines above, so I felt it would be a repetition. Please let me know if you still want me to add it.